src/plugins/intel_cpu/tests/functional/single_layer_tests/classes/reduce.cpp

```diff
@@ -49,7 +49,15 @@ std::string ReduceCPULayerTest::getTestCaseName(testing::TestParamInfo<ReduceLay
         result << "KeepDims=false_";
     result << "netPRC=" << netPrecision << "_";
     result << "inPRC=" << inPrc << "_";
-    result << "outPRC=" << outPrc << "_";
+    result << "outPRC=" << outPrc;
+
+    if (!config.empty()) {
+        result << "_PluginConf";
+        for (const auto& configItem : config) {
+            result << "_" << configItem.first << "=";
+            configItem.second.print(result);
+        }
+    }
 
```

Comment:
Just an idea: maybe we can add new helper to [CommonTestUtils](https://github.com/openvinotoolkit/openvino/blob/master/src/tests/test_utils/common_test_utils/include/common_test_utils/common_utils.hpp) and resuse it anywhere?